gen/calc.py

```diff
@@ -410,8 +418,8 @@ def validate_os_type(os_type):
             'aws': {
                 'must': {'rexray_config_contents': yaml.dump(AWS_REXRAY_CONFIG)},
             },
-            'empty': {
-                'must': {'rexray_config_contents': yaml.dump('')},
+            'disabled': {
```

Comment:
This is in the Enterprise docs: https://docs.mesosphere.com/1.7/administration/installing/custom/configuration-parameters/

I'd vote to just leave the old name, makes the docs simpler.